Corax.sln

```diff
@@ -0,0 +1,173 @@
+﻿
```

Comment:
2nd sln file is a bit problematic four our CI. Can we merge that to RavenDB.sln?

Reply:
Is it back? I have killed it. Let me check

Reply:
Forgot to push that one 🤦‍♂️